apps/common/src/python/mediawords/languages/en/en_stop_words.txt

```diff
@@ -1,4175 +1,798 @@
-meaning
-meaningful
-meanings
 means
-meant
 meantime
 meanwhile
-measure
-measured
-measurement
-measures
-measuring
-meat
-mechanical
-mechanism
-medical
-medicine
-medium
-meet
-meeting
-meetings
-meets
-melody
-melting
 member
 members
-membership
-memory
 men
-mental
-mention
-mentioned
-merchant
-merchants
-mere
 merely
-merger
-merit
-mess
-message
-messrs
-met
-meta
-metal
-method
-methods
-mg
-middle
-middle-class
-midnight
 might
+might've
 mightn't
-mighty
-mile
-miles
-military
-milk
-milligrams
-million
-millions
-mind
-minds
+mightnt
+mil
+mill
 mine
-mines
-minimal
-minimum
-minor
-minority
 minus
-minute
-minutes
-mirror
 miss
-missed
-missile
-missing
-mission
-mistake
-mix
-mixed
-mixture
-mlle
-mm
-mme
-mobile
-mode
-model
-moderate
-modern
-modest
-mold
-molecule
-moment
-monday
-money
-month
-monthly
-months
-monument
-mood
-moon
-moral
-morality
 more
 moreover
-morning
 most
 mostly
-mother
-mothers
-motion
-motive
-motives
-motor
-mount
-mountain
-mounted
-mouth
 move
-moved
-movement
-movements
-moves
-movie
-movies
-moving
 mr
 mrs
-ms
-msgr
-mss
-mst
 much
-mud
-multiple
-multiply
-municipal
-murder
-muscle
-muscles
-music
-musical
-musician
-musicians
+mug
 must
-mustard
+must've
 mustn't
-mutual
+mustnt
 my
+myse"
 myself
-mysterious
-mystery
-myth
+myse”
+mz
 n
-naked
 name
-named
 namely
-names
-narrative
-narrow
-nation
-national
-natural
-naturally
-nature
-naval
-nd
+nay
 near
-nearby
-nearest
 nearly
-neat
 necessarily
 necessary
-necessity
-neck
-need
-needed
-needing
 needn't
-needs
-negative
-negotiations
-neighbor
-neighborhood
-neighboring
-neighbors
+neednt
 neither
-nerves
-nervous
-nest
-net
-network
-neutral
 never
 neverf
 neverless
 nevertheless
-new
-newer
-newest
-newly
-news
-newspaper
-newspapers
 next
-nice
-night
-nights
 nine
-nineteenth
 ninety
-ninth
 no
 no-one
-noble
 nobody
-nodded
-noise
 non
 none
 nonetheless
-noon
 noone
 nor
-normal
-normally
-norms
-north
-nose
+nos
 not
-notable
-note
 noted
-notes
 nothing
-notice
-noticed
-notion
 notwithstanding
-noun
-nov
-novel
-novels
-november
 now
 nowhere
-nude
+null
 number
 numbers
-numeral
-numerous
-nuts
 o
-object
-objective
-objectives
-objects
-obligations
-obliged
-observation
-observations
-observe
-observed
-observers
 obtain
 obtained
-obvious
 obviously
-occasion
-occasional
-occasionally
-occasions
-occupation
-occupied
-occur
-occurred
-occurrence
-occurring
-occurs
-ocean
-oct
-october
-odd
 of
 off
-offer
-offered
-offering
-offers
-office
-officer
-officers
-offices
-official
-officials
 often
 oh
-oil
 ok
 okay
-okt
-old
-older
-oldest
 on
 once
 one
 one's
 ones
 only
-onset
 onto
-op
-open
-opened
-opening
-openly
 opens
-operate
-operated
-operating
-operation
-operational
-operations
-operator
-opportunities
-opportunity
-opposed
-opposite
-opposition
-optimal
 or
-oral
-orchestra
 ord
-order
-ordered
-ordering
-orderly
-orders
-organ
-organic
-organization
-organizations
-organized
-origin
-original
-originally
+org
```

Comment:
TLD (again, we might have to remove it afterwards but let's just skip it on the first pass to see what we get with a very non-restrictive list of stopwords).